src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -4831,6 +4832,152 @@ void UpdateIntermediateSurfaceHeatBalanceResults(EnergyPlusData &state, Optional
     }
 }
 
+void UpdateNonRepresentativeSurfaceResults(EnergyPlusData &state, Optional_int_const ZoneToResimulate)
+{
+    int firstZone = 1;
+    int lastZone = state.dataGlobal->NumOfZones;
+
+    if (present(ZoneToResimulate)) {
+        firstZone = ZoneToResimulate;
+        lastZone = ZoneToResimulate;
+    }
+
+    for (int zoneNum = firstZone; zoneNum <= lastZone; ++zoneNum) {
+        // Heat transfer surfaces
+        int firstSurf = state.dataHeatBal->Zone(zoneNum).HTSurfaceFirst;
+        int lastSurf = state.dataHeatBal->Zone(zoneNum).HTSurfaceLast;
+        for (int surfNum = firstSurf; surfNum <= lastSurf; ++surfNum) {
+            auto &surface(state.dataSurface->Surface(surfNum));
+            int repSurfNum = surface.RepresentativeCalcSurfNum;
+
+            if (surfNum != repSurfNum) {
+#if 0
```

Comment:
@nealkruis What's the status on this PR? If we're trying to get this in to v9.6, the `#if endif` needs to be removed?

Reply:
@mjwitte this is ready to merge in my opinion. The #if 0 code is harmless, but useful for debugging.

Reply:
Yeah the `#if` in there is a bit weird.  If these are valuable for debugging issues, and issues may occur, then this should just be in the extra warnings section maybe.  Or something.